src/main/kotlin/lotto/business/ReceivedAmount.kt

```diff
@@ -0,0 +1,17 @@
+package lotto.business
+
+class ReceivedAmount(val amount: Int) {
+
+    init {
+        require(amount >= LOTTO_PRICE) { "로또 구입 금액은 ${LOTTO_PRICE}원 이상이어야 합니다." }
+        require(amount % LOTTO_PRICE == 0) { "로또 구입 금액은 ${LOTTO_PRICE}원 단위로 입력해주세요." }
+    }
+
+    fun getTicketCount(): Int {
+        return amount / LOTTO_PRICE
+    }
+
+    companion object {
+        const val LOTTO_PRICE = 1000
```

Comment:
숫자 3자리 단위로 `_`를 붙일 수 있다는 사실 알고 계셨나요? 😃
구분자 덕분에 숫자의 단위를 조금 더 파악하기 쉬워서 저는 자주 애용하는 편이에요. 😉

```suggestion
        const val LOTTO_PRICE = 1_000
```